src/main/webapp/trip/tripScript.js

```diff
@@ -0,0 +1,28 @@
+var date = new Date();
+var tdate = date.getDate();
+var month = date.getMonth() + 1;
+var year = date.getUTCFullYear();
+if(tdate < 10){
+    tdate = '0' + tdate;
+}
```

Comment:
Can you add a comment here. Something like
"Prefix single digit date/month with 0 so they has a consistent 2 digit format."